[#2158] Add More Documentation for Title Component (#2159)
reposense/RepoSense: docs/ug/customizingReports.md

</box>

3\. Add a git `.mailmap` file at the top-level of the repository, specifying mapped authors/commiters and/or e-mail addresses as per [gitmailmap documentation](https://git-scm.com/docs/gitmailmap). Any mappings specified here will be applied by git before all other RepoSense configurations. Configuration via `.mailmap` is particularly useful if you want the mapping to apply for all git commands as well instead of just for RepoSense.

<!-- ------------------------------------------------------------------------------------------------------ -->

> ### Add a Title

Reviewer: ```suggestion
### Add a title
```

A title component can be added by creating a file titled `title.md` in the assets directory. You can specify the assets directory according to the reference below:
  {{ embed("Appendix: **CLI syntax reference**", "cli.md") }}

> The title can render a combination of HTML and plaintext ([example](title.md)), and will appear on the top of the left panel as shown below:

Reviewer: The example title file here opens up within the user guide documentation itself as another page. Instead, it should appear as code, perhaps pointing to the code file in GitHub. Can you change the link here accordingly?

> The title can render a combination of HTML and plaintext ([example](title.md)), and will appear on the top of the left panel as shown below:

Reviewer: We can indicate that the title uses Markdown syntax explicitly.

```suggestion
The title can render a combination of Markdown/HTML and plaintext ([example](title.md)), and will appear on the top of the left panel as shown below:
```

![Title Component Example](../images/title-example.png)

Do note that the width of the title is bound by the width of the left panel.

> For more information on how to create a Markdown file, see the [Markdown Guide](https://www.markdownguide.org/).

Reviewer: ```suggestion
For more information on how to use Markdown, see the [Markdown Guide](https://www.markdownguide.org/).
```
